cron/format/json.go

```diff
@@ -128,6 +145,7 @@ func AsJSON2(r *pkg.ScorecardResult, showDetails bool,
 		},
 		Date:     r.Date.Format("2006-01-02"),
 		Metadata: r.Metadata,
+		AggScore: jsonFloatScore(score),
```

Comment:
AggregateScore in full might be better to avoid confusion.

Reply:
sure. Note this is not the JSON field, but jut the structure field. The JSON field is currently called just `score`. Is that ok?